manifests/kustomize/base/argo/minio-artifact-secret.yaml

```diff
@@ -5,5 +5,6 @@ data:
 kind: Secret
 metadata:
   name: mlpipeline-minio-artifact
-  namespace: kubeflow
+  labels:
+    application-crd-id: kubeflow-pipelines
```

Comment:
Looks like this needs to be added to every resource.

Can we add it by kustomization.yaml like `commonLabels` in https://github.com/kubernetes-sigs/kustomize/blob/master/examples/helloWorld/README.md#staging-kustomization?

Reply:
thanks. done